### DISCUSSION TITLE
Django tenants with ASGI not working

### DISCUSSION THREAD

**Discussion by suhail-chand** [django-tenants/django-tenants#981] [Q&A]
I had setup a multi-tenant system using django-tenants, but I am getting an error when I setup ASGI using Daphne as 'FakeTenant' object has no attribute 'domain_subfolder'
Can anyone please help understanding this error.

**Comment by aayushbasnet012** [comment]
Hey, did you find any solution to this problem. I am facing the same problem with this.
